src/main/java/com/nike/cerberus/command/core/ViewConfigCommand.java

```diff
@@ -22,7 +22,7 @@
 import com.nike.cerberus.operation.Operation;
 import com.nike.cerberus.operation.core.ViewConfigOperation;
 
-import static com.nike.cerberus.command.cms.CreateCmsClusterCommand.COMMAND_NAME;
+import static com.nike.cerberus.command.core.ViewConfigCommand.COMMAND_NAME;
```

Comment:
Are we importing static from the same class?

Reply:
Same as above